README.md

```diff
@@ -425,6 +425,9 @@ such a scenario.
 | CLI Option  | Description       |
 |-------------|-------------------|
 | `--reporter-cli-silent`         | The CLI reporter is internally disabled and you see no output to terminal. |
+
+| `--reporter-cli-show-timestamp` | This prints the local time for each request made. | 
```

Comment:
shouldn't this be called `--reporter-cli-show-timestamps` 🤔 , thoughts?
Or, can this change be just part of existing `--verbose` flag?

Reply:
1. I couldn't decide between choosing `timestamp` and `timestamps`. I went with the former, but since you pointed it out I guess it feels out of place for someone reading it the first time. I'll change it.


2. Since this is a CLI reporter specific option I feel `--timestamps` may be a bit confusing as other flags with `--` work on all reporters.